src/styles/main.scss

```diff
@@ -0,0 +1,62 @@
+/* #region Normalize */
+body {
+  margin: 0;
+}
+/* #endregion */
+
+/*#region Variables */
+$pin-size: 24px;
+/* #endregion */
+
+/* #region Mixins */
+
+@mixin position {
+  position: absolute;
+  left: 50%;
+  bottom: 50%;
+}
```

Comment:
do not make mixins that do not take in any arguments. use placeholder instead

Reply:
Ok. Added an argument to the mixin.